dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
+    project: assembl
+
+
+mount=source:
+    bind: .
+    path: /opt/assembl
+
+mount=wheels:
+    bind: wheels/
+    path: /opt/wheels
+
+image=faber:
+    image: assembl-faber
+    context: docker/
+    dockerfile: Dockerfile.fabric
+
+#
+# Create a database image with the correct user and fixtures loaded
+#
+job=setup-postgres:
+    use: faber
+    mounts: [source, wheels]
+    net-mode: '{project}_default'
+    command: "fab devenv setup_postgresql_docker"
+    depends: [empty-database]
+
+compose=empty-database:
+    files: [docker/compose/empty-db.yaml]
+    project: '{project}'
+
+#
+# Build local wheels so that dev dependencies don't need to be installed in the 
+# runtime image.
+#
+image=wheeler:
+    image: assembl-wheeler
+    context: docker/
+    dockerfile: Dockerfile.wheel
+
+job=build-wheels:
```

Comment:
[Python wheels](http://pythonwheels.com/) has all the relevant info summarized pretty well.

The advantage in this PR is that you don't need to install dev dependencies in the webapp container. 

This makes it much faster to rebuild for two reasons. One is fewer ubuntu packages to install, and the second is installing from a wheel is significantly faster than installing from source.